src/content/app/genome-browser/state/track-config/trackConfigSlice.ts

```diff
@@ -68,52 +98,190 @@ export const updateTrackStatesAndSave: ActionCreator<
   browserStorageService.saveTrackStates(trackStates);
 };
 
+export const getTracksStateInfo = (
+  genomeId: string,
+  trackCategories: GenomeTrackCategory[]
+) => {
+  const tracksInfo: TrackConfigs = {};
+
+  tracksInfo['gene-focus'] = {
+    showSeveralTranscripts: false,
+    showTranscriptIds: false,
+    showTrackName: false,
+    showFeatureLabel: true,
+    trackType: TrackType.GENE
+  };
+
+  trackCategories.forEach((category) => {
+    category.track_list.forEach((track) => {
+      const trackId = track.track_id.replace('track:', '');
```

Comment:
Why are we doing this? We do not control track ids that come from the backend. I see you are modifying the track ids and use the modified ids when saving the state to browser storage. Why?

Reply:
We will do this in another ticket.